### PR TITLE
Schedule refactor

### PR DESCRIPTION
**Pull request by swopl**
Has some refactors
Maybe don't merge this yet, just check
Won't be available to change these for a while
Will do sched refactor later. If im not lazy

### CODE REVIEW

**Review by KatarinaM-SV6 — APPROVED** [review]
good refactoring

File: HospitalSystem.ConsoleUI/Director/RenovationUI.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+{
+    public RenovationUI(Hospital hospital, List<Room> loadedRooms) : base(hospital, loadedRooms)
+    {
+
+    }
+
+    public override void Start()
+    {
+        throw new NotImplementedException("Whoops");  // TODO: don't have such an inheritance, it bad
+    }
```
Comment (VladimirGolosin) [inline HospitalSystem.ConsoleUI/Director/RenovationUI.cs:15]: Shouldn't this be implemented?

**Review by VladimirGolosin — APPROVED** [review]
Seems fine to me.

File: HospitalSystem.ConsoleUI/Director/RenovationUI.cs (hunk 1 above)
Comment (swopl) [reply HospitalSystem.ConsoleUI/Director/RenovationUI.cs:15]: Nah, won't hook into this one in this way, only using its public funcs. Will do in a better way in future

**Pull request merged**